### DISCUSSION TITLE
CIS benchmark - OS Level

### DISCUSSION THREAD

**Discussion by iamaravindragu** [aquasecurity/trivy#7280] [Q&A]
### Question

Dear Team , 
  
    Will trivy support **OS Level CIS benchmarking** report for Ubuntu 22.04 and Amazon Linux 2023 . If yes Kindly share me the documentation links . I can see CIS benchmarking for K8s and for Amazon Account . Is that possible for Operating Systems ? 

--
Aravindan R 

### Target

Filesystem

### Scanner

None

### Output Format

None

### Mode

None

### Operating System

Ubuntu 22.04 and Amazon linux 2023

### Version

```bash
Latest Version
```

**Comment by itaysk** [comment]
Hi, as of now there are no plans to do this
